doc/start/quick-ceph-deploy.rst

```diff
@@ -293,7 +297,7 @@ example::
    ``rados put`` command on the command line. For example::
 
      echo {Test-data} > testfile.txt
-     ceph osd pool create mytest
+     ceph osd pool create mytest {number-of-placement-groups}
```

Comment:
You might considerer setting autoscaling??
`ceph osd pool set foo pg_autoscale_mode on`
Makes life a lot easier.

Reply:
How would that work in the context of the quick start instructions?

It seems that specifying a pg_num is required:

> [cephDeployUser@mon-node1 ~]$ sudo ceph osd pool create mynewtest
Invalid command: missing required parameter pg_num(<int[0-]>)

And turning on autoscaling when creating a group doesn't seem to be a valid command, either:

> [cephDeployUser@mon-node1 ~]$ sudo ceph osd pool create mynewtest pg_autoscale_mode on
Invalid command: pg_autoscale_mode doesn't represent an int

Have I missed something here?

Reply:
Perhaps the solutions is to create it with pg=0,
I can do the following:
```
# ceph mgr module enable pg_autoscaler
# ceph osd pool create mynewtest 0
# ceph osd pool set mynewtest pg_autoscale_mode on
# ceph osd pool autoscale-status
POOL        SIZE TARGET SIZE RATE RAW CAPACITY  RATIO TARGET RATIO BIAS PG_NUM NEW PG_NUM AUTOSCALE
mynewtest     0               3.0       23420G 0.0000               1.0      8            on
```

Notice that ceph-mgr/pg_autoscaler set PGs to 8.
perhaps a better approach to creating pools?

Reply:
That's a good idea. Thanks for your feedback and suggestion; it is appreciated.

I've submitted an amended commit to include enabling PG autoscaling.

Reply:
Marking conversation as 'resolved', but happy to re-open if there is further feedback or comments.